pkg/engine/lifecycletest/fuzzing/README.md

```diff
@@ -66,3 +66,55 @@ By default, reproduction test files will be written to a temporary directory. A
 specific directory can be configured using the
 `PULUMI_LIFECYCLE_TEST_FUZZING_REPRO_DIR` environment variable.
 :::
+
+### Running a suite of fuzz tests
+
+The `TestFuzz` test generates and tests a set of fixtures, and can be run with
+the top-level `test_lifecycle_fuzz` target:
+
+```
+make test_lifecycle_fuzz GO_TEST_RACE=false
+```
+
+`GO_TEST_RACE=false` is passed to disable race detection. You can omit this (and
```

Comment:
Should fuzz tests turn this off by default?

Reply:
Good shout. I'll do that.